rebar.config

```diff
@@ -1,9 +1,6 @@
 %% -*- mode: erlang;erlang-indent-level: 2;indent-tabs-mode: nil -*-
 %% ex: ts=4 sw=4 ft=erlang et
 
-%% == Erlang Compiler ==
-{minimum_otp_vsn, "23"}.
```

Comment:
We have no proof that this project can run on OTP21, @vkatsuba.
Because our tools only verify it on OTP23+, we can only assert that it runs on OTP23+.
I would accept a PR if…
1. You add a note to the README stating that we only provide guarantees that the system runs on OTP23+ since that's what we're testing it in, but the `minimum_otp_vsn` is `"21"` because X and Y systems required it.
2. Instead of removing this, you replace it with `"21"`.

Reply:
Done.